### ISSUE TITLE
Test installation procedure with Ubuntu 20.04

### ISSUE THREAD

**Issue by pierrepo** [plasmabio/plasma#128]
When Ubuntu 20.04 will be available on VM images, it would be nice to test the installation procedure on the OS.

**Comment by jtpio** [comment]
It would indeed be interesting to try on the latest 20.04 once it's available.

For now OVH only offers the following:

![image](https://user-images.githubusercontent.com/591645/79975676-a8804f80-849b-11ea-82b3-fe80977dba9b.png)

I've been testing on 19.10 for a while and it seems to be working fine.

**Comment by pierrepo** [comment]
Yes. There's no rush. This issue is just a reminder.

**Comment by jtpio** [comment]
It looks like the 20.04 have started to be available:

![image](https://user-images.githubusercontent.com/591645/80471683-09989f00-8944-11ea-81ff-f6ec5269d623.png)

After doing a quick test, it appears that `cockpit-docker` is not longer available on 20.04:

- https://cockpit-project.org/blog/cockpit-215.html
- https://bugs.launchpad.net/ubuntu/+source/cockpit/+bug/1875028

> cockpit-docker is being deprecated in favor of cockpit-podman, and will not be maintained upstream. We have stopped building it for Ubuntu ≥ 20.04, Debian ≥ 11, and Fedora >= 32 (i. e. the current development series), but will keep it for the stable releases (i. e. backports).

**Comment by pierrepo** [comment]
> It looks like the 20.04 images have started to be available

Cool!

> it appears that cockpit-docker is no longer available on 20.04

Ouch. `cockpit-docker` on 18.04 and `cockpit-podman` on 20.04, not very convenient...

**Comment by jtpio** [comment]
Apart from that the rest seems to be working well.

`cockpit-podman` doesn't seem to be available on 20.04 either at the moment.

**Comment by pierrepo** [comment]
>Apart from that the rest seems to be working well.

:tada:

> cockpit-podman doesn't seem to be available on 20.04 either at the moment.

:cry: dependency hell

**Comment by pierrepo** [comment]
Is it possible to install `cockpit-docker` only for ubuntu < 20.04 est print a warning message for ubuntu >= 20.04 (until `cokpit-podman` is available)?

**Comment by jtpio** [comment]
Indeed, we'll need to do something like that to support both.

**Comment by jtpio** [comment]
It sounds like the simplest for now would be to install `cockpit-docker` from the deb directly, as recommended here:

 - https://bugs.launchpad.net/ubuntu/+source/cockpit/+bug/1875028
- https://launchpad.net/ubuntu/+source/cockpit/215-1~ubuntu19.10.1/+build/18889196/+files/cockpit-docker_215-1~ubuntu19.10.1_all.deb

Also `cockpit-podman` is still not available on Ubuntu 20.04:

```bash
$ sudo apt install cockpit-podman
Reading package lists... Done
Building dependency tree
Reading state information... Done
E: Unable to locate package cockpit-podman
```
